src/Nethermind/Nethermind.State/IWorldState.cs

```diff
@@ -108,9 +108,9 @@ public interface IWorldState : IJournal<Snapshot>, IReadOnlyStateProvider
 
     /* snapshots */
 
-    void Commit(IReleaseSpec releaseSpec, bool isGenesis = false, bool commitStorageRoots = true);
+    void Commit(IReleaseSpec releaseSpec, bool isGenesis = false, bool commitRoots = true);
 
-    void Commit(IReleaseSpec releaseSpec, IWorldStateTracer? tracer, bool isGenesis = false, bool commitStorageRoots = true);
+    void Commit(IReleaseSpec releaseSpec, IWorldStateTracer? tracer, bool isGenesis = false, bool commitRoots = true);
```

Comment:
I think this is not the first change in the `IWorldState.Commit`. Should we consider flags maybe for the options, or do you think that `bool commitRoots` will stick for a while and are the only toggle that will be passed down?

Reply:
Hopefully last change 😉

Mostly just renamed param, as also now effects stateroot